azure-storage-cpp-lite/include/http/libcurl_http_client.h

```diff
@@ -148,6 +148,16 @@ namespace microsoft_azure {
                 return m_input_stream;
             }
 
+            void set_absolute_timeout() override {
+                check_code(curl_easy_setopt(m_curl, CURLOPT_TIMEOUT, 30L)); // Timeout of 30 seconds
```

Comment:
I think for operations like get_blob_properties we should set this to 5 second. 30 second for those atomic operations is too long

Reply:
I don't know, 30 seconds is fairly standard for us.  It shouldn't be common, but if we get unlucky and hit a partition move, more than 5 seconds can happen

Reply:
ok - if it is standard across other things, let's keep it

Reply:
Well okay - isn't it actually faster to time out at 5-10 seconds, and then let it retry ? :)